src/Microsoft.DotNet.Darc/src/Darc/Options/AddBuildToChannelCommandLineOptions.cs

```diff
@@ -16,10 +16,10 @@ internal class AddBuildToChannelCommandLineOptions : CommandLineOptions
         [Option("channel", Required = true, HelpText = "Channel to assign build to.")]
         public string Channel { get; set; }
 
-        [Option("source-branch", HelpText = "Branch that should be used as base for the promotion build.")]
+        [Option("source-branch", HelpText = "Branch that should be used as base for the promotion build. Required if source-sha is specified.")]
         public string SourceBranch { get; set; }
```

Comment:
I believe there's a way to automatically handle the fact that both options are required from here

Reply:
Nevermind, doesn't looks like that exists.